lib/widgets/message_list.dart

```diff
@@ -81,6 +82,16 @@ class _MessageListState extends State<MessageList> {
     final length = model!.messages.length;
     assert(model!.contents.length == length);
     return StickyHeaderListView.builder(
+        // TODO: Offer `ScrollViewKeyboardDismissBehavior.interactive` (or
+        //   similar) if that is ever offered:
+        //     https://github.com/flutter/flutter/issues/57609#issuecomment-1355340849
+        keyboardDismissBehavior: Platform.isIOS
+            // This seems to offer the only way to close the keyboard on iOS.
+            // It's not ideal; see TODO above.
+            ? ScrollViewKeyboardDismissBehavior.onDrag
```

Comment:
only *built-in* way — as the name of the other option, `manual`, suggests, we could also build such a way ourselves with a gesture recognizer, and IIRC that issue thread has several people saying they did just that (while wishing Flutter would provide a built-in way instead)